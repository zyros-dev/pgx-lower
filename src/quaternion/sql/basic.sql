CREATE TABLE test(id SERIAL, quat QUATERNION);
INSERT INTO test(quat) VALUES ('(1,2,3,4)');
SELECT * FROM test;

