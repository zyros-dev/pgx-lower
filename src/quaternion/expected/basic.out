CREATE TABLE test(id SERIAL, quat QUATERNION);
INSERT INTO test(quat) VALUES ('(1,2,3,4)');
SELECT * FROM test;
 id |   quat    
----+-----------
  1 | (1,2,3,4)
(1 row)

